src/libraries/System.Private.DataContractSerialization/src/System/Xml/XmlConverter.cs

```diff
@@ -30,10 +30,10 @@ internal static class XmlConverter
         public const int MaxUInt64Chars = 32;
         public const int MaxPrimitiveChars = MaxDateTimeChars;
 
-        private static UTF8Encoding s_utf8Encoding;
-        private static UnicodeEncoding s_unicodeEncoding;
+        private static UTF8Encoding? s_utf8Encoding;
```

Comment:
I wonder how much of this type overlaps with XmlConverter in System.Private.Xml

Reply:
It looks like it wraps XmlConvert. See:

https://github.com/dotnet/runtime/blob/aa5fdab9654d74bc6274c0b5d820272c8e859621/src/libraries/System.Private.DataContractSerialization/src/System/Xml/XmlConverter.cs#L125-L134